Tests/Clappr_Tests/Classes/Plugin/Container/ContainerTests.swift

```diff
@@ -353,21 +353,71 @@ class ContainerTests: QuickSpec {
                     expect(didResizeValue?.height).toEventually(equal(10))
                 }
             }
+            
+            describe("#reload") {
+                context("options") {
+                    it("changes source URL") {
+                        Loader.shared.resetPlaybacks()
+                        Loader.shared.register(playbacks: [StubPlayback.self])
+                        let container = Container()
+                        container.load(Source.invalid)
+                        
+                        container.setNeedsReload()
+                        container.reload()
```

Comment:
A `container.play()` would reflect better the behavior here. What do you think?

Reply:
Done